snippets/app/build.gradle

```diff
@@ -52,3 +55,16 @@ dependencies {
     // [END maps_android_places_upgrade_snippet]
 }
 // [END maps_android_places_install_snippet]
+
+secrets {
+    // To add your Google Maps Platform API key to this project:
+    // 1. Create or open file secrets.properties in the root folder of the project, which will be
+    // read by secrets_gradle_plugin
+    // 2. Add this line, replacing YOUR_API_KEY with a key from a project with Places API enabled:
+    //        PLACES_API_KEY=YOUR_API_KEY
+    // 3. Add this line, replacing YOUR_API_KEY with a key from a project with Maps SDK for Android
```

Comment:
Since the snippets here don't require the Maps SDK, can we omit #3?

Reply:
Done